packages/phoenix/src/utils.rs

```diff
@@ -42,24 +40,42 @@ pub struct StakeInitInfo {
     pub min_reward: i128,
 }
 
+#[contracttype]
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct LiquidityPoolInitInfo {
+    pub admin: Address,
+    pub lp_wasm_hash: BytesN<32>,
+    pub share_token_decimals: u32,
+    pub swap_fee_bps: i64,
+    pub fee_recipient: Address,
+    pub max_allowed_slippage_bps: i64,
+    pub max_allowed_spread_bps: i64,
+    pub token_init_info: TokenInitInfo,
+    pub stake_init_info: StakeInitInfo,
+}
+
 #[cfg(test)]
 mod tests {
     use super::*;
 
     #[test]
     fn test_validate_int_parameters() {
         // The macro should not panic for valid parameters.
-        validate_int_parameters!(1, 2, 3).unwrap();
-        validate_int_parameters!(1, 1, 1).unwrap();
-        validate_int_parameters!(1i128, 2i128, 3i128, Some(4i128), None::<i128>).unwrap();
-        validate_int_parameters!(None::<i128>, None::<i128>).unwrap();
-        validate_int_parameters!(Some(1i128), None::<i128>).unwrap();
+        validate_int_parameters!(1, 2, 3);
+        validate_int_parameters!(1, 1, 1);
+        validate_int_parameters!(1i128, 2i128, 3i128, Some(4i128), None::<i128>);
+        validate_int_parameters!(None::<i128>, None::<i128>);
+        validate_int_parameters!(Some(1i128), None::<i128>);
+    }
 
-        validate_int_parameters!(1, -2, 3).unwrap_err();
-        validate_int_parameters!(0, 1, 3).unwrap_err();
-        validate_int_parameters!(1, 1, 0).unwrap_err();
-        validate_int_parameters!(Some(0i128), None::<i128>).unwrap_err();
-        validate_int_parameters!(Some(-1i128), None::<i128>).unwrap_err();
+    #[test]
+    #[should_panic(expected = "value cannot be less than or equal zero")]
+    fn should_panic_when_value_less_than_or_equal_zero() {
+        validate_int_parameters!(1, -2, 3);
+        validate_int_parameters!(0, 1, 3);
+        validate_int_parameters!(1, 1, 0);
+        validate_int_parameters!(Some(0i128), None::<i128>);
+        validate_int_parameters!(Some(-1i128), None::<i128>);
```

Comment:
Actually this test doesn't work now as intended, since it will stop at first panic - then should_panic is applied and test passes. 
Doesn't matter that much, but probably each of those cases should have a separate should_panic test case.